.github/workflows/phase_1_python.yml

```diff
@@ -33,11 +42,13 @@ jobs:
         run: |
           /tmp/trivy image \
             --format cyclonedx \
+            --pkg-types os \
```

Comment:
would this option still pick up Python packages installed in the container?

Reply:
I haven't double checked that, but this is my assumption.